precompiles/precompile_test.go

```diff
@@ -188,7 +188,7 @@ func TestPrecompilesPerArbosVersion(t *testing.T) {
 	log.SetDefault(log.NewLogger(glogger))
 
 	expectedNewMethodsPerArbosVersion := map[uint64]int{
-		0:  89,
+		0:  91,
```

Comment:
We should probably just ignore ArbDebug in this test since it's not on prod chains